templates/cdk.master.auth-webhook.py

```diff
@@ -226,8 +236,21 @@ def forward_request(json_req, url):
     return False
 
 
-@app.route('/{{ api_ver }}', methods=['POST'])
-def webhook():
+def ack(req, **kwargs):
+    # Successful checks will set auth and user data in the 'req' dict
+    log_secret(text='ACK', obj=req)
+    return aiohttp.web.json_response(req, **kwargs)
+
+
+def nak(req, **kwargs):
+    # Force unauthenticated, just in case
+    req.setdefault('status', {})['authenticated'] = False
+    log_secret(text='NAK', obj=req)
+    return aiohttp.web.json_response(req, **kwargs)
+
+
+@routes.post('/{api_ver}')
```

Comment:
```suggestion
@routes.post('/{{ api_ver }}')
```

Reply:
Flask uses the double braces, but aiohttp uses single braces: https://docs.aiohttp.org/en/latest/web_quickstart.html#aiohttp-web-variable-handler

Reply:
It wasn't `{{ api_ver }}` for flask.  It was because k8s-master [sets api_ver to be rendered](https://github.com/charmed-kubernetes/charm-kubernetes-master/blob/master/reactive/kubernetes_master.py#L1069) as a template var.

Perhaps we thought we'd have different webhooks for different authn (or at least handle them differently within the same webhook).  Perhaps we thought we'd have some other path listener on the same port.  Perhaps we didn't want to spam the auth-webhook log if an errant service started hammering `$ip:5000/$not_api_ver`.  I dunno.  But it wasn't double braced for flask.

Reply:
Ha!  My mistake.